app/models/concerns/foreman_openscap/assetmix.rb

```diff
@@ -0,0 +1,7 @@
+module ForemanOpenscap
+  module Assetmix
```

Comment:
could you please rename to something more meaningful? e.g. BelongsToAssetMixin, also we tend to use ActiveSupport::Concern